uswds/uswds-site
# USWDS-Site - Guidance: Update usability guidance for Date Picker and Date Range Picker

# Summary

Date Picker and Date Range Picker should always allow users to type the date manually if that is preferred.
Add text to the Usability Guidance sections for both components to let users know they should account for this preference.

## Related issue

Closes #2294  

## Preview links

Preview links:

[Date Picker 
](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cb-date-picker-and-date-range-guidance/components/date-picker/#usability-guidance)

[Date Range Picker](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cb-date-picker-and-date-range-guidance/components/date-range-picker/#usability-guidance)

## Changelog links:

[Date Picker 
](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cb-date-picker-and-date-range-guidance/components/date-picker/#latest-updates)

[Date Range Picker](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cb-date-picker-and-date-range-guidance/components/date-range-picker/#latest-updates)

## Problem statement

Not all users want to use the calendar picker for dates. The Date Picker and the Date Range Picker components should be implemented so that users can enter dates manually if they choose to do so.

## Solution

Update the guidance for Date Picker and Date Range Picker so that users are aware that these components should be implemented to allow users to type dates in manually.
 
## Major changes

N/A, this is a change to the usability guidance.

## Testing and review

1. Click on each supplied federalist link.
2. Ensure that this text appears in "usability guidance" after any previous supplied guidance:

Always allow a user to type in the date manually. Usability testing suggests some people prefer manually typing the date rather than using the calendar picker. Whenever possible, keep the keyboard active so people can enter in the date information without having to use the picker.

Before opening this PR, make sure you’ve done whichever of these applies to you:
